spec/features/tests/show_view_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+feature "user views show page", %{
+  As a user
+  I want to view a food truck's information
+  so that I can learn more about it
+} do
+  # Acceptance Criteria
+  # * If a user goes to the show page, they see a list of food trucks.
+
+  scenario "user views show page of a food truck" do
+    sample_food_truck_1 = FoodTruck.create(
+      name: "Chicken & Rice Guys",
+      description: "Food truck that specializes in chicken and lamb shawarma.",
+      avg_rating: 4,
+      location: "Harvard Square"
+    )
+
+    visit food_truck_path(sample_food_truck_1.id)
```

Comment:
You should be able to access the path just by passing in the object; no need for `.id`.